### ISSUE TITLE
Provide cross platform app

### ISSUE THREAD

**Issue by fkromer** [asyncapi/studio#172]
#### Reason/Context

Many backend devs are not using Node.Js and are forced to install Node.js to beeing able to use the app locally.

#### Description

Would be great if one could simply use a Tauri or Electron based app.

**Comment by derberg** [comment]
@fkromer wouldn't it be easier just to provide a docker image for it?

**Comment by fkromer** [comment]
@derberg Right, a docker image on DockerHub would address this issue as well.

**Comment by magicmatatjahu** [comment]
This issue is connected to the https://github.com/asyncapi/studio/issues/30 Maybe in the future we will think about electron app, but not at the moment, we are focusing on delivery plugins, spec builder (by forms), better visualiser, better linter etc.

**Comment by magicmatatjahu** [comment]
I will mark that issue as `long-term` enhancement, but atm we can use studio from docker image:

```bash
docker run -it -p 8000:80 asyncapi/studio
```

Thanks @fgreinacher! 👏🏼
